src/mbgl/style/layers/heatmap_layer_properties.hpp

```diff
@@ -24,7 +24,8 @@ struct HeatmapIntensity : PaintProperty<float> {
     static float defaultValue() { return 1; }
 };
 
-using HeatmapColor = ColorRampProperty;
+struct HeatmapColor : ColorRampProperty {
```

Comment:
Could you please provide more details for this change? I'm supposing this is to address some compiler error.